### DISCUSSION TITLE
Matching Python if-else

### DISCUSSION THREAD

**Discussion by elbaro** [ast-grep/ast-grep#1299] [Q&A] [answered]
Can ast-grep extract a scoped block from if-elif-else or try-catch-finally?

I would like to replace

```py
if A:
   ...
elif B:
  XX
else C:
  ...
```

with

```py
XX
```

But `sg -l py -p "if some_statement: $$$A else: $$$B" -r $$$A` does not match any code.

**Comment by HerringtonDarkholme** [comment]
Please use the playground to debug the pattern https://ast-grep.github.io/playground.html

**Reply by elbaro** [reply]
![image](https://github.com/ast-grep/ast-grep/assets/1851290/0acb57ad-c478-472a-bcae-2561bd17f818)
Work in the playground but how can I use the multiline pattern in CLI?

**Reply by HerringtonDarkholme** [reply, marked as answer]
you can add newline in you cli argument. 
Please search the internet.
https://stackoverflow.com/questions/38642312/bash-how-to-pass-a-line-feed-to-a-script
